src/Views/batchprocessor.html

```diff
@@ -163,6 +173,15 @@ <h3>Select Basin Characteristics</h3>
 
         <!-- Start batch status -->
         <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
+            <!-- Start instructions -->
+            <div id="batchStatusInstructions"> To check batch status: <br>

```

Comment:
id isn't necessary since it isn't being used anywhere

```suggestion
            <div> To check batch status: <br>

```